### PR TITLE
Complete functionailty of TrelloSync V1

### PR DESCRIPTION
**Pull request by jebbinSCL**
Note: Main changes are in Import.fs
All other changes are just updating the logging to use Serilog.

### CODE REVIEW

**Comment by pallenSL** [comment]
It's rather inconsistent when logging.  We have "Card: <card>" and "Card : <card>" and "Card <card>".

**Comment by jebbinSCL** [comment]
Logging has been updated.

File: TrelloSync/TrelloSync/Import.fs [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        |> Option.map (fun x -> x.Profile.Id)
+    
+    let prepSessionAndCorrData (adminProfiles : ProfileAndTrelloId []) (speakerProfiles : ProfileAndTrelloId []) (card : CardWithCorrespondence) = 
+        let adminId = 
+            match card.TrelloCard.AdminId with
+            | Some id -> findSrmProfileId id adminProfiles
+            | None -> None
+        
+        let speakerId = findSrmProfileId card.TrelloCard.CardId speakerProfiles |> Option.get
+
```
Comment (lclarksl) [inline TrelloSync/TrelloSync/Import.fs]: Is there a reason why admin id is an Option and speaker id isn't?

File: TrelloSync/TrelloSync/Import.fs [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        let correspondence = 
+            // If the card doesn't have a admin email or speakerEmail, then we can't complete all the required information for the correspondence item, so the correspondence won't be processed and inserted. 
+            if Array.isEmpty card.Comms then [||]
+            else 
+                match card.TrelloCard.AdminEmail with
+                | Some adminEmail -> 
+                    match card.TrelloCard.SpeakerEmail with
+                    | "" -> [||]
+                    | speakerEmail -> 
+                        let aId = Option.get adminId
```
Comment (lclarksl) [inline TrelloSync/TrelloSync/Import.fs]: Simlarly, any reason admin email is treated as an Option and speaker email not?

File: TrelloSync/TrelloSync/Import.fs (hunk 1 above)
Comment (jebbinSCL) [inline TrelloSync/TrelloSync/Import.fs]: The reason for this and below is the same: the card must have a speaker, but the admin may not have been assigned. I'll modify the code to make this a bit more obvious.

**Comment by jebbinSCL** [comment]
Code updated for to make difference between the speaker id and admin id on a card clearer.

**Comment by lclarksl** [comment]
:shipit:

**Comment by pallenSL** [comment]
:shipit:

**Pull request merged**